data/mac-database/README.md

```diff
@@ -0,0 +1,12 @@
+# Building the MAC address database
+
+The MAC address database can be created using the `build-database.py` script. The script will download the latest registered MAC address block information from [https://macaddress.io](https://macaddress.io) and creates a SQLITE database file.
+
+The generated database is read performance optimized and tried to keep as small as possible for searching MAC address OUIs (Organizationally Unique Identifiers) blocks and returning the registered company name.
+
+    $ python3 build-database.py
+
+The final database will be named `mac-addresses.db`.
+
+In nymea the `MacAddressDatabase` will search by default for this database file in `/usr/share/nymea/mac-addresses.db` and provides an asynch threaded mechanism to get the company name for a given MAC address.
```

Comment:
not sure about the `${XDG_DATA_DIRS}`, also, it would point to `/usr/local/share`... and it's nothing "locally" installed. it's shared data availabe for the system. Also, starting as user would make it load from a different location, where it wouldn't be available.

Reply:
XDG_DATA_DIRS will find it in /usr/share/... Always, even if started as user... Don't confuse it with XDG_DATA_HOME